tests/conftest.py

```diff
@@ -46,3 +57,28 @@ def testbed(request):
 
     tbinfo = TestbedInfo(tbfile)
     return tbinfo.testbed_topo[tbname]
+
+@pytest.fixture(autouse=True)
+def loganalyzer(duthost, request):
+    loganalyzer = LogAnalyzer(ansible_host=duthost, marker_prefix="loganalyzer")
+    # Add start marker into DUT syslog
+    marker = loganalyzer.init()
+    yield loganalyzer
+    if not request.config.getoption("--disable_loganalyzer") and not pytest.disable_loganalyzer:
+        # Read existed common regular expressions located with legacy loganalyzer module
+        loganalyzer.load_common_config()
+        result = loganalyzer.analyze(marker)
+        if not result:
+            pytest.fail("Log analyzer failed.")
+        if result["total"]["match"] != 0:
+            pytest.fail("Found errors: {}".format(result))
```

Comment:
Can you pretty-print the result? Based on current experience with ansible loganalyzer I find it very hard to find in logs loganalyzer result.